src/components/googleLogInBtn.jsx

```diff
@@ -10,17 +11,31 @@ class GoogleLogInBtn extends Component {
     var googleId = googleUser.getId();
 
     let profile = googleUser.getBasicProfile();
-    firebase
+    var stringPart = profile.getEmail().split('@');
+    var userInfo = firebase.database().ref('users/' + localStorage.getItem('id'));
+    if (userInfo === null)
+    {
+      firebase
       .database()
       .ref("users/" + googleId)
-      .set({
+      .update({
         fullName: profile.getName(),
         firstName: profile.getGivenName(),
         lastName: profile.getFamilyName(),
         email: profile.getEmail(),     
+        profileURL: googleUser.profileObj.imageUrl,
+        id: googleUser.getId(),
+        userName: '',
+        bio: ''
+
       });
-      
+    }
+   
       this.props.signInState(true, googleId);
+      this.props.setProfilePic(googleUser.profileObj.imageUrl);
+      localStorage.setItem('id', googleId)
```

Comment:
we dont need these bc we pass googleId through signInState already which will shoot up to local storage